test/testdata/workflow/persistence/by_service/02-sonataflow_platform.yaml

```diff
@@ -21,7 +21,7 @@ spec:
     template:
       buildArgs:
       - name: QUARKUS_EXTENSIONS
-        value: org.kie.kogito:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT,org.kie.kogito:kogito-addons-quarkus-persistence-jdbc:999-SNAPSHOT,io.quarkus:quarkus-jdbc-postgresql:3.2.9.Final,io.quarkus:quarkus-agroal:3.2.9.Final
+        value: org.kie.kogito:kogito-addons-quarkus-persistence-jdbc:999-SNAPSHOT,io.quarkus:quarkus-jdbc-postgresql:3.2.9.Final,io.quarkus:quarkus-agroal:3.2.9.Final
```

Comment:
I guess we can add these values by default when persistence is on. @jordigilh can you please open a follow-up issue? I'll do an assessment.

Reply:
Will do